src/encoded/schemas/experiment_series.json

```diff
@@ -0,0 +1,211 @@
+    "id": "/profiles/experiment_series.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": [ "related_datasets"],
+    "identifyingProperties": ["uuid", "accession", "aliases"],
+    "additionalProperties": false,
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/accession" },
+        { "$ref": "mixins.json#/aliases" },
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/submitted" },
+        { "$ref": "mixins.json#/notes" },
+        { "$ref": "mixins.json#/submitter_comment" },
+        { "$ref": "series.json#/properties" }
+    ],
+    "dependencies": {
+        "status": {
+            "oneOf": [
+                {
+                    "required": ["date_released"],
+                    "properties": {
+                        "status": {
+                            "enum": ["released", "revoked"]
+                        }
+                    }
+                },
+                {
+                    "not": {
+                        "properties": {
+                            "status": {
+                                "enum": ["released", "revoked"]
+                            }
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "accession": {
+            "accessionType": "SR"
+        },
+        "award": {
+            "default": "5854a1dc-4c50-44b3-ba58-0a6f4f399297",
+            "linkEnum": [
+                "5854a1dc-4c50-44b3-ba58-0a6f4f399297"
+            ],
+            "permission": "import_items"
+        },
+        "date_released": {
+            "title": "Date released",
+            "comment": "Do not submit, value is assigned whe the object is releaesd.",
+            "type": "string",
+            "anyOf": [
+                {"format": "date-time"},
+                {"format": "date"}
+            ],
+            "permission": "import_items"
+        },
+        "lab": {
+            "default": "cb0ef1f6-3bd3-4000-8636-1c5b9f7000dc",
+            "linkEnum": [
+                "cb0ef1f6-3bd3-4000-8636-1c5b9f7000dc"
+            ],
+            "permission": "import_items"
+        },
+        "related_datasets": {
+            "minItems": 1
+        },
+        "status": {
+            "title": "Status",
+            "type": "string",
+            "permission": "import_items",
+            "default": "in progress",
+            "enum" : [
+                "in progress",
+                "submitted",
+                "released",
+                "archived",
+                "deleted",
+                "replaced",
+                "revoked"
+            ],
+            "permission": "import_items"
```

Comment:
This repeats the "permission" line a few lines above.

Reply:
Repeat? This one is for status. Others are for other properties.

Reply:
This is a repeat of "permission": "import_items" line a few lines above. VSCode's JSON linter marks it.

Reply:
Sorry. Saw that. Thanks.